### DISCUSSION TITLE
Cannot connect to Slingbox

### DISCUSSION THREAD

**Discussion by larsenv** [GerryDazoo/Slinger#410] [General]
Slinger can't find the Slingbox Solo on my network. giving the `Can't find a slingbox on network. Please make sure it's plugged in and connected. Check config.ini`, `Giving up, sorry...` message.

I connected it to my router, and I can see the MAC on there. It's possible that the Slingbox password could have been reset and I pressed the reset button while trying to figure this out with no luck.

I tried SlingPlayer 2 on Windows and SlingPlayer 1 on Mac to see if it could find the Slingbox, but it couldn't.

What should I do?

Thanks.

**Comment by GerryDazoo** [comment]
Seeing the MAC doesn't help. What is it's IP address? To probe for slingboxes the software sends out a UDP broadcast message on port 5004 and expects a reply from all the slingboxes connected to your LAN.  If my code and the sling code can't find it then your got some networking issues or maybe the solo was configured with a static IP that's not on your LAN.
PS. Next time please post all the output from the server. There's useful information there,

**Comment by larsenv** [comment]
On my router it doesn't list an IP address and a tool to scan the LAN doesn't list the Slingbox. I turned on UPnP on the router which didn't fix it. My router is one leased by Spectrum that is honestly not that good. You can only configure it with a phone. I've used this Slingbox before and it worked before. Do you know what I should try?

**Comment by larsenv** [comment]
Just read that Sling was bricking some Solo units with a firmware update. Could that be why it can't connect to anything? Yesterday was the first time that I plugged it in in quite some time.
